src/Payments.Mvc/Controllers/PdfController.cs

```diff
@@ -26,7 +26,7 @@ public PdfController(ApplicationDbContext dbContext, IStorageService storageServ
         }
 
         [HttpGet("/pdf/{id}")]
-        public async Task<ActionResult> Invoice(string id)
+        public async Task<Microsoft.AspNetCore.Mvc.ActionResult> Invoice(string id)
```

Comment:
Strange that you need the namespace preamble here -- are there other `Controller` and `ActionResult` that are causing conflicts?